salt/metalk8s/map.jinja

```diff
@@ -24,23 +41,145 @@
 }, merge=defaults.get('metalk8s')) %}
 
 {% set kubernetes = salt['grains.filter_by']({
-  'default': {}
-}, merge=defaults.get('kubernetes')) %}
+  'Ubuntu': {}
+}, grain='os', merge=defaults.get('kubernetes')) %}
```

Comment:
Why this change ? We want to merge this variable also for RedHat. If you have nothing special to override here keep the old one please :)